File: JavaScript projects/JavaScript Basics/For Loop/Exercises/06_oscars.js
function oscars(input){
    let name = input[0];
    let points = Number(input[1]);
    let countJudges = Number(input[2]);
    
    for (let i = 3; i < input.length; i += 2){
        points += input[i].length * Number(input[i+1]) / 2;
        
        if (1250.5 < points){
            console.log(`Congratulations, ${name} got a nominee for leading role with ${(points).toFixed(1)}!`);
            return;
        }
    }
    
    
    console.log(`Sorry, ${name} you need ${(1250.5 - points).toFixed(1)} more!`);
    
}
oscars(["Zahari Baharov","205",4,"Johnny Depp","45","Will Smith","29","Jet Lee","10","Matthew Mcconaughey","39"]);

File: Python projects/Python Basics/04_for_loops/exercises/06_oscars.py
actors_name = str(input())
academy_points = float(input())
number_judges = int(input())
total_points_actor = academy_points

for judges in range(number_judges):
    name_judge = str(input())
    points_judge = float(input())
    if total_points_actor > 1250.5:
        # print(f"Congratulations, {actors_name} got a nominee for leading role with {точки}!")
        break
    total_points_actor += len(name_judge) * points_judge / 2

if total_points_actor > 1250.5:
    print(f"Congratulations, {actors_name} got a nominee for leading role with {total_points_actor:.1f}!")
else:
    print(f"Sorry, {actors_name} you need {1250.5 - total_points_actor:.1f} more!")
